scripts/jenkins/cloud/ansible/group_vars/all/versioned.yml

```diff
@@ -48,6 +48,9 @@ versioned_features:
     enabled: "{{ when_cloud9 }}"
   ardana-ssh-keyscan:
     enabled: "{{ when_cloud9 }}"
+  crowbar_sync_mark_timeout_update:
+    enabled: "{{ when_cloud9 }}"
+    value: "2.0"
```

Comment:
Do we really need that much more? Have you checked how much time was actually needed in those failing runs?

Reply:
@skazi0 it was my recommendation that we use a brutal 2.0 for maximum effect. I don't want people to spend time tweaking this value over and over and I see no downside to using a bigger value.